template/web/app/paymaster-bundler/_contracts/usePaymasterBundlerContract.ts

```diff
@@ -0,0 +1,17 @@
+import { baseSepolia } from 'viem/chains';
+import { generateContractHook } from '@/hooks/contracts';
+import { entryPoint } from '../constants';
+import { PaymasterBundlerABI } from './PaymasterBundlerABI';
+
+/**
+ * Returns contract data for the PaymasterBundler contract.
+ */
+export const usePaymasterBundlerContract = generateContractHook({
+  abi: PaymasterBundlerABI,
+  [baseSepolia.id]: {
+    chain: baseSepolia,
+    address: entryPoint,
```

Comment:
@0xAlec is this the correct constant to use here?

Reply:
updated